templates/engage/index.html

```diff
@@ -63,7 +63,7 @@ <h1>Engage pages</h1>
         <select name="tag-selector" id="tag-selector" class="u-no-margin--bottom">
           <option value="all">All tags</option>
           {% for tag in tags %}
-            <option value="{{ tag }}">{{ tag }}</option>
+            <option value="{{ tag }}">{% if tag == "" %}-{% else %}{{ tag }}{% endif %}</option>
```

Comment:
```suggestion
            <option value="{{ tag }}">{% if tag == "" %}-{% else %}{{ tag }}{% endif %}</option>
```
Can you add the `disabled` attribute if the value is `-`